We are replacing the included pip wheel with a newer one without
certifi related security vulnerabilities.

Not intended for upstream.

--- Python-3.9.19/Lib/ensurepip/__init__.py
+++ Python-3.9.19/Lib/ensurepip/__init__.py
@@ -12,7 +12,7 @@ from . import _bundled
 
 __all__ = ["version", "bootstrap"]
 _SETUPTOOLS_VERSION = "58.1.0"
-_PIP_VERSION = "23.0.1"
+_PIP_VERSION = "24.0"
 _PROJECTS = [
     ("setuptools", _SETUPTOOLS_VERSION, "py3"),
     ("pip", _PIP_VERSION, "py3"),
